pkg/reconciler/ingress/resources/secret.go

```diff
@@ -97,6 +98,10 @@ func targetWildcardSecretName(originSecretName, originSecretNamespace string) st
 }
 
 func makeSecret(originSecret *corev1.Secret, name, namespace string, labels map[string]string) *corev1.Secret {
+	if pkg.ShouldFilterByCertificateUID() {
```

Comment:
I think we should always copy the CertificateUID regardless if we enable filtering or not

Maybe update `MakeTargetSecretLabels` and drop this conditional?